rmgpy/molecule/element.py

```diff
@@ -319,6 +319,8 @@ def getElement(value, isotope=-1):
 #Bond Dissociation Energies
 #Reference: Huheey, pps. A-21 to A-34; T.L. Cottrell, "The Strengths of Chemical Bonds," 2nd ed., Butterworths, London, 1958; B. deB. Darwent, "National Standard Reference Data Series," National Bureau of Standards, No. 31, Washington, DC, 1970; S.W. Benson, J. Chem. Educ., 42, 502 (1965).
 #(C,C,1.5) was taken from an unsourced table that had similar values to those used below, should be replaced if a sourced value becomes available
+#(C,C,2.5) is C#C - (CbenzeneC - C-C)
+elements = ['C','N','H','O','S','Cl','Si']
```

Comment:
It's the list of elements supported by the BDE dict

Reply:
It tells extension generation what atoms are feasible to use in groups.py.

Reply:
Could you mention that in a comment? Or possibly rename the variable to be more descriptive?

The main consideration is so that future developers know that they should update this list if the BDEs get updated, or they should not update this list without updating BDEs.